scripts/check_formatting

```diff
@@ -50,6 +50,9 @@ find scripts -type f -exec shellcheck {} +
 # if formatting has to be applied.
 cargo fmt --all -- --check
 
+# Check formatting for experimental Rust crates.
+cargo fmt --all --manifest-path=experimental/Cargo.toml -- --check
```

Comment:
Oh! Cool. I guess I have to add the same thing for `examples`, and `oak_abi` to the script.